.travis.yml

```diff
@@ -3,8 +3,27 @@ notifications:
   email: false
 before_script:
   - npm prune
-after_success:
-  - npm run semantic-release
-branches:
-  except:
-    - /^v\d+\.\d+\.\d+$/
+
+# Run on multiple versions of Node
+# Will run the default stage multiple times
+node_js:
+  - '8'
+  - '9'
```

Comment:
I'd prefer we leave this off, otherwise `.nvmrc` won't be used in CI, which means dev and CI versions could drift.